.mypy.ini

```diff
@@ -2,60 +2,34 @@
 # Global Options
 
 [mypy]
-files = python/smqtk, tests
+# Files/Directories to investigate.
+files = smqtk_image_io, tests
 
 # Disable caching
 incremental = False
 
-
-###############################################################################
-# Options for different parts of SMQTK
-
-[mypy-smqtk.*]
 # Disallows defining functions without type annotations or with incomplete type
 # annotations.
-;disallow_untyped_defs = True
-
-# Disallows defining functions with incomplete type annotations
-disallow_incomplete_defs = True
-
-# Type-checks the interior of functions without type annotations.
-check_untyped_defs = True
+disallow_untyped_defs = True
 
-# Allows variables to be redefined with an arbitrary type, as long as the
-# redefinition is in the same block and nesting level as the original
-# definition.
-allow_redefinition = True
+###############################################################################
+# Options for different parts of SMQTK-Indexing
 
-[mypy-tests.*]
-# Disallows defining functions with incomplete type annotations
-disallow_incomplete_defs = True
+;[mypy-smqtk_indexing.*]
 
-# Incrementally making sub-modules more strict
-[mypy-smqtk.iqr.iqr_session]
-disallow_untyped_defs = True
-[mypy-smqtk.utils.plugin]
-disallow_untyped_defs = True
-[mypy-tests.utils.test_plugin]
-disallow_untyped_defs = True
+;[mypy-tests.*]
 
 ###############################################################################
 # TPL-module options
 
-[mypy-numpy.*]
+[mypy-numpy]
```

Comment:
This might be OK for now, however the `.*` part means to apply to any import of numpy or its sub-modules. If something imports a sub-module now the "doesn't support typing" error will be thrown again. Technically no sub-modules are being used right now but there's no reason to not include recursive sub-modules (it's not the case I believe that any of them selectively support typing).